Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM stardog-eps-docker.jfrog.io/stardog:6.1.2
+FROM stardog-eps-docker.jfrog.io/stardog:6.2.4
```

Comment:
do we want this any higher?

Reply:
Is there a higher one for v6? This is the highest release for that major version that I saw in the release notes.

Reply:
oh yeah, maybe not. I was going off the binaries in artifactory, where I see 6.2.7